app/models/ext_management_system.rb

```diff
@@ -271,6 +274,7 @@ def connection_configurations=(options)
 
     delete_unused_connection_configurations(options)
   end
+  alias connection_definitions= connection_configurations=
```

Comment:
> To solve this problem, this PR introduce a new method connection_definitions that return the struct connection_configurations= expects ... so we can do GET connection_definitions and POST to connection_configurations=

This doesn't seem right to me. Having two differently named attributes for the same data seems confusing. Can you find a way to have the getter/setter both present/accept the data in the form that you need? This seems important enough to me to go into a PR of its own with specs that test this behavior thoroughly at the model level.